app/src/main/java/com/example/placefinder/dto/Park.kt

```diff
@@ -2,8 +2,19 @@ package com.example.placefinder.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class Park (@SerializedName("parkCode") var parkCode: String, var parkName: String) {
+/**
+ * A data class representing a Park
+ *
+ * @property parkCode The code of the park
+ * @property parkName The name of the park
+ */
+data class Park (@SerializedName("parkCode") var parkCode: String, @SerializedName("parkName") var parkName: String) {
```

Comment:
I have added the "@SerializedName" annotation before the "parkName" attribute. This will make sure that the parkName attribute gets directly matched up with the JSON object called "parkName".

Reply:
@SerializedName is only required if the name of the fields in the JSON field differ from the name of the attributes in the data class to which they are being mapped.  That is not the case here.